velox/runner/tests/CMakeLists.txt

```diff
@@ -23,3 +23,20 @@ target_link_libraries(
   velox_parse_parser
   velox_parse_expression
   GTest::gtest)
+
+velox_add_library(velox_query_replay_runner QueryReplayRunner.cpp)
+
+velox_link_libraries(
```

Comment:
See my comments in the other PR, I assume this is stacked but doesn't show up properly in GH?